spec/models/miq_product_feature_spec.rb

```diff
@@ -4,7 +4,7 @@
 
 describe MiqProductFeature do
   before do
-    @expected_feature_count = 872
+    @expected_feature_count = 878
```

Comment:
@jrafanie right, it's usually only thing that conflicts for me with other PRs. Though it's a job for another PR, @martinpovolny maybe you could do it as the part of features tests?